### PR TITLE
feat: add Contribute guide

### PR DESCRIPTION
**Pull request by aLIEzsss4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,54 @@ pnpm dev
...
+
+Thank you for supporting AutoChessia! We look forward to having you on board!
+ [join our Discord]( https://discord.gg/Qget5JQHtr).
+
+### Local development setup
+
+!!!
+The following steps are only necessary if you want to contribute to AutoChessia. To use AutoChessia in your project, install the [packages](#packages) from pnpm 
+
+1. Install go (required to build [packages/services](packages/services/)): [https://go.dev/doc/install](https://go.dev/doc/install)
```
Comment (noyyyy) [inline README.md:100]: Why do we need go?

**Review by noyyyy — CHANGES_REQUESTED** [review]
Contribution guide and code of conduct should be a single file and linked to README. Refer https://github.com/HelheimLabs/autochessia/community to add standard.

File: README.md (hunk 1 above)
Comment (aLIEzsss4) [reply README.md]: remove go 、grpc

File: README.md [hunk 2]
```diff
@@ -81,6 +81,50 @@ pnpm dev
...
+This repo contains all the source code of AutoChessia, and we welcome pull requests at any time. If you have any ideas for new features or modifications, we are happy to discuss them with you.
+
+Your feedback is very important to us. Whether it's code contributions or product suggestions, we sincerely want to hear your thoughts. Let's work together to make AutoChessia more powerful and user-friendly!
+
+If you encounter any issues during code contribution or usage, please feel free to reach out to us as well.
+
+Thank you for supporting AutoChessia! We look forward to having you on board!
+ [join our Discord]( https://discord.gg/Qget5JQHtr).
+
+### Local development setup
```
Comment (noyyyy) [inline README.md:95]: There's some overlap here with `get start`. Merge them to chapter.

Comment (aLIEzsss4) [reply README.md:95]: i think get start doesn't conflict with contributions

Comment (noyyyy) [reply README.md:95]: > i think get start doesn't conflict with contributions

@aLIEzsss4 Not agree. Writing document for how to start this project is a part of contribution guide. You can view the opensource guide website quickly before continue this pr. https://opensource.guide/

**Pull request closed without merging**